api/test/nostd/list_test.cc

```diff
@@ -130,3 +131,43 @@ TEST(ListTest, PopFront)
 
   EXPECT_EQ(int_list.front(), 1);
 }
+
+// Tests that after using the assignment operator, changing the new 
+// doesn't change the original
+TEST(ListTest,AssignmentOperatorDoesNotChangeOriginal)
+{
+
+  list<int> first_list = {0, 1, 2, 3, 4};
+  list<int> first_list_copy = {0, 1, 2, 3, 4};
+  
+  list<int> second_list = {5, 6, 7, 8, 9};
+  second_list = first_list;
+  second_list.pop_back();
+  second_list.pop_back();
+  second_list.pop_back();
+
+  EXPECT_EQ(first_list == first_list_copy, true);
```

Comment:
I think you can do
  EXPECT_EQ(first_list, first_list_copy);
  EXPECT_NE(first_list, second_list);
since list::operator== is defined, but if not then
  EXPECT_TRUE(...) and EXPECT_FALSE(...) are preferable to EXPECT_EQ(foo, true/false).

Reply:
Switched to EXPECT_TRUE and EXPECT_FALSE.